sauerkraut.md

```diff
@@ -0,0 +1,28 @@
+# Sauerkraut
+
+## Einleitung
+
+Dies ist ein einfaches Familienrezept für Sauerkraut nach russischer Art.
+
+## Zutaten
+
+- 3kg Weißkohl, möglichst reif
+- 1 große Möhre (ca. 160g)
+- 1 gehäufter Esslöffel grober Salz
+
+## Zubereitung
+
+- Weißkohl putzen und mit einem großen Messer in dünne, längliche Streifen schneiden.
+- Möhre schälen und raspeln.
+- Alle Zutaten in eine große Schüssel geben und vorsichtig und gründlich vermischen. In diesem Schritt noch keine Kraft aufwenden.
+- Das Kraut in ein großes Glas drücken. In diesem Schritt ordentlich Kraft aufwenden.
+- Das Glas bei Raumteperatur 3 Tage offen stehen lassen.
+- Während dieser Zeit sollte sich oben Flüssigkeit ansammeln. Mit einem Spieß das Kraut immer wieder mal vorsichtig lockern, so dass Gase austreten können und die Flüssigkeit absickert.
+- Nach 3 Tagen das Glas mit einem Deckel verschließen und kühl lagern.
```

Comment:
Wie gut die Fermentation funktioniert hängt vom Kohl und der Umgebungstemperatur ab. Am besten nach Geschmack entscheiden ob es schon sauer genug erscheint, dann kann es in den Kühlschrank. Sonst kann es auch bis zu 2 Wochen draussen bleiben.